content/developer/reference/frontend/services.rst

```diff
@@ -131,6 +133,48 @@ Reference List
    * - :ref:`user <frontend/services/user>`
      - provides some information related to the current user
 
+.. _frontend/services/company:
+
+Company service
+---------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `company`
+* Dependencies: `user`, `router`, `cookie`
+
+Odoo has a concept of companies. You can manage several with the same Odoo instance. 
```

Comment:
i don't think you should use the word `instance`. it feels like a odoo word that is wrong.

Maybe rewrite the first lines like this:

```
Odoo allows users to manage more than one companies from the same database.
```